pkg/api/message/v1/service.go

```diff
@@ -99,28 +85,14 @@ func (s *Service) Close() {
 	if s.ctxCancel != nil {
 		s.ctxCancel()
 	}
-	s.subDispatcher.Shutdown()
-
-	if s.nc != nil {
-		s.nc.Close()
-	}
 
 	s.wg.Wait()
 	s.log.Info("closed")
 }
 
 func (s *Service) HandleIncomingWakuRelayMessage(msg *wakupb.WakuMessage) error {
 	env := envelopes.BuildEnvelope(msg)
-
-	envB, err := pb.Marshal(env)
-	if err != nil {
-		return err
-	}
-
-	err = s.nc.Publish(envelopes.BuildNatsSubject(env.ContentTopic), envB)
-	if err != nil {
-		return err
-	}
+	s.subDispatcher.HandleEnvelope(env)
```

Comment:
I have absolutely no idea why we were publishing to `nats` here. Seems completely useless. But it's old code that I didn't write, so I hope I'm not missing something super subtle.